ramls/holdingsrecordssource.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A holdings records source",
+  "type": "object",
+  "properties": {
```

Comment:
I think this record needs the `metadata` property in order for the change metadata to be recorded and available to the client.

An example of this property can be found in the holdings record.

Reply:
Added the metadata property.